WRIO-InternetOS/js/jsonld/image.js

```diff
@@ -0,0 +1,30 @@
+var Image = function(opts) {
```

Comment:
@balthazzar I would suggest using ES6 classes here, because wrio-internetOS is backed by babel, and would be great to see it in new code. Your opinion on that?